.github/workflows/electrs.yml

```diff
@@ -0,0 +1,175 @@
+name: Electrs
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      path-filter: ${{ steps.filter.outputs.path-filter }}
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'pull_request'
+
+      - uses: dorny/paths-filter@v2
+        if: github.event_name == 'pull_request'
+        id: filter
+        with:
+          filters: |
+            path-filter:
+              - './!((doc|infrastructure|)/**)'
+
+  build-publish-electrs-initcontainer:
+    needs: detect-changes
+    if: |
+      github.event_name != 'pull_request'
+        || needs.detect-changes.outputs.path-filter == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Load environment variables
+        if: github.event_name == 'workflow_dispatch'
+        uses: keep-network/load-env-variables@v1
+        with:
+          environment: 'ropsten'
```

Comment:
> Please put a comment explaining why are we using `ropsten` here. Just like we have in other repos:
> 
> ```
>           # TODO: Consider passing of `environment` input instead of using 
>           # hardcoded value. Would require some rework in action's code or
>           # in config files.
> ```

Done in https://github.com/keep-network/electrs/pull/1/commits/0a9c81686bbfbf573428f1c83594f3bcec4498c1.

Reply:
Done in 0a9c816.